redbot/cogs/mod/casetypes.py

```diff
@@ -103,4 +103,10 @@
         "image": "\N{SPEAKER WITH CANCELLATION STROKE}",
         "case_str": "Voice Kick",
     },
+    {
+        "name": "warning",
+        "default_setting": True,
+        "image": "\N{WARNING SIGN}",
+        "case_str": "Warning",
+    },
```

Comment:
This needs to be put in `redbot/core/generic_casetypes.py` + it needs a variation selector for the emoji (i.e. `"\N{WARNING SIGN}\N{VARIATION SELECTOR-16}"`).

Reply:
Hopefully got this fixed and working. I assume what I put is what you were recommending. Never used the Variation selector before so just copied from here.